client/src/App.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import Layout from './Components/Common/Layout/index';
+
+const App = ({ store }) => (
+  <Provider store={store}>
```

Comment:
@UjwalBattar why do we have a sotre here and on the iclient/src/ndex.js file? just want to make sure.

Reply:
So here, we are pulling the app store and destructuring to use in our client side